xbmc/platform/linux/AppParamParserLinux.cpp

```diff
@@ -52,6 +52,17 @@ CAppParamParserLinux::CAppParamParserLinux() : CAppParamParser()
 
 CAppParamParserLinux::~CAppParamParserLinux() = default;
 
+void CAppParamParserLinux::Parse(char* argv[], int nArgs)
```

Comment:
Why not a CAppParamParserWebOS that extends CAppParamParserLinux?

Reply:
Done!